integration-cli/docker_cli_build_test.go

```diff
@@ -2293,6 +2293,29 @@ func TestBuildExposeOrder(t *testing.T) {
 	logDone("build - expose order")
 }
 
+func TestBuildExposeHostPort(t *testing.T) {
```

Comment:
@duglin 
Since added code use:
fmt.Fprintf(b.OutStream,  "Dockerfile EXPOSE (ip:hostPort:containerPort) has been deprecated"
Both buildImageWithStdoutStderr and buildImageWithOut could only get a blank output for me.